packages/cli/test/unit/commands/promote/index.test.ts

```diff
@@ -154,28 +179,49 @@ describe('promote', () => {
       // say "yes" to the prompt
       client.stdin.write('y\n');
 
+      await expect(client.stderr).toOutput(
+        `Successfully created new deployment of ${projectName} at ${inspectorUrl}`
+      );
       await expect(exitCodePromise).resolves.toEqual(0);
+      expect(createWasCalled).toBe(true);
     });
 
     it('should promote a preview deployment with --yes', async () => {
+      const inspectorUrl = `https://vercel.com/user/${projectName}/abcdefghijlmo`;
+
+      let createWasCalled = false;
       const { cwd, previousDeployment } = initPromoteTest({
         deploymentTarget: 'preview',
       });
+
+      client.scenario.post(
```

Comment:
Placed the scenario in-line because I wanted to assert the exact body submitted as well.